docs/pages/guides/configuration-management/configuration-management.md

```diff
@@ -60,7 +66,7 @@ mesheryctl pattern apply BookInfoApp
 
 This will apply the pattern BookInfoApp, which has already been imported into Meshery.
 
-See [mesheryctl pattern subcommand section](../reference/mesheryctl/#cloud-native-pattern-configuration-and-management) for more details on the `pattern` subcommand.
+See [mesheryctl pattern subcommand section](../../reference/mesheryctl/#cloud-native-pattern-configuration-and-management) for more details on the `pattern` subcommand.
```

Comment:
```suggestion
See [mesheryctl pattern subcommand section]({{ site.baseurl }}/reference/mesheryctl/#cloud-native-pattern-configuration-and-management) for more details on the `pattern` subcommand.
```

Reply:
The link should be pointing to [Cloud Native Pattern Configuration and Management](https://docs.meshery.io/reference/mesheryctl#cloud-native-pattern-configuration-and-management) // @iArchitSharma